src/App.css

```diff
@@ -2,6 +2,10 @@
   text-align: left;
 }
 
+.marked {
+  background-color: red;
+}
```

Comment:
Good use of CSS to add a highlight to a row, to make this even better try to make your class names something that anyone reading them could work out what it does, i.e. "highlighted-row" or similar